Final Legal Chatbot Files/malegislature_primary_law_extraction.py

```
@@ -0,0 +1,80 @@
+    "https://malegislature.gov/Laws/GeneralLaws/PartI/TitleXX/Chapter139",
+    "https://malegislature.gov/Laws/GeneralLaws/PartI/TitleXV/Chapter93A"
+]
+
+headers = {
+    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36"
+}
+
+# Function to scrape a single chapter
+def scrape_chapter(chapter_url):
+    print(f"🔍 Scraping Chapter: {chapter_url}")
+    chapter_response = requests.get(chapter_url, headers=headers)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)